financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/FinancialConnectionsSheetNativeViewModel.kt

```diff
@@ -161,7 +182,7 @@ internal class FinancialConnectionsSheetNativeViewModel @Inject constructor(
      */
     fun onResume() = viewModelScope.launch {
```

Comment:
Do we still need the coroutine scope now?

Reply:
yep! still needed for the mutex